app/src/main/java/net/aiscope/gdd_app/ui/mask/customview/MaskLayer.kt

```diff
@@ -91,6 +89,14 @@ class MaskLayer(private val imageMatrix: Matrix) {
     fun initSize(size: Size) {
         require(!sizeInitialized()) { "Size was initialized already!" }
         this.size = size
+
+        currentStateBitmap = Bitmap.createBitmap(
+            size.width,
+            size.height,
+            Bitmap.Config.ARGB_8888
+        )
+
+        currentStateBitmapCanvas = Canvas(currentStateBitmap)
```

Comment:
Doesn't seem to matter, I think, but I was just curious why did you move these initialisations down here.

Reply:
@eduardb Well, previously we had a lazily initialized `val currentStateBitmap`, that just happened to be initialized (used) after the `size` var had been set. 

Since we moved it to a `lateinit var` it made sense to me to just initialize the variable when the size gets set.